### ISSUE TITLE
GUIGame replay replay

### ISSUE THREAD

**Issue by wknowleskellett** [jarhill0/MoonLander#30]
The recreation of a computer-generated game can only be played once - pressing enter to play it again runs a game with no input
worthwhile to fix?

**Comment by jarhill0** [comment]
This is true. Considering we've received grades for the class, fix if you want! I think the fix would basically require adding a "rewind" method to the bit buffer.

I'm considering trying to compile for iOS, but I haven't gotten around to it.
